cli_reference/index.adoc

```diff
@@ -0,0 +1,38 @@
+[id="cli-tools-overview"]
+= {product-title} CLI tools overview
+include::modules/common-attributes.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user performs different tasks while working on {product-title} which includes a range of operations such as the following:
+
+* Managing clusters
+* Building, deploying, and managing applications
+* Managing deployment processes
+* Developing Operators
+* Creating and maintaining Operator catalogs
+
+{product-title} offers a set of command-line interface (CLI) tools that simplify these tasks by enabling users to perform various administration and development operations from the terminal.
+These tools expose simple commands to manage the applications, as well as the lower level tools to interact with each component of the system.
```

Comment:
s/...as well as the lower level tools to interact with each component of the system./...as well as interact with each component of the system.
Not sure what 'lower level tools' mean in this context.